app/controllers/item_categories_controller.rb

```diff
@@ -17,6 +17,7 @@ def show
   # GET /item_categories/new
   def new
     @item_category = ItemCategory.new
+    @item_category.status = 'Active'
```

Comment:
I think `@item_category = ItemCategory.new(status: 'Active')` would have been a better way to do this.